src/twisted/web/wsgi.py

```diff
@@ -535,6 +537,9 @@ class WSGIResource:
     An L{IResource} implementation which delegates responsibility for all
     resources hierarchically inferior to it to a WSGI application.
 
+    Note that the WSGI response environ includes a non-standard variable
+    REMOTE_PORT.
+
```

Comment:
It's a bit hard for me to review this as I have never worked with WSGI app

I am reading some docs from https://docs.twisted.org/en/stable/web/howto/web-in-60/wsgi.html

Based on that, maybe we can have it like this.

I think that `Note that` is kind of redendand for API docs.
All the API doc content is a `note that` thing :)

```suggestion
    The C{environ} argument passed to the application, includes the C{REMOTE_PORT} key to complement the C{REMOTE_ADDR} key.

```